osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -119,6 +126,16 @@ public enum GlobalAction
         [Description("Toggle mute")]
         ToggleMute,
 
+        //Song selector bindings
+        [Description("Choose a random song")]
+        FooterRandom,
+
+        [Description("Change beatmap options")]
+        FooterOptions,
+
+        [Description("Select mods")]
+        FooterMods,
```

Comment:
To expand on the above comment, doing this appears to work but actually blocks music controller keybindings in song select (previous song and play/pause) because of a binding conflict. This not what we want.

Reply:
Thanks for the great feedback!I'll get on this right away.